src/main/java/com/microsoft/azure/vmagent/AzureVMAgentTemplate.java

```diff
@@ -428,7 +430,8 @@ public AzureVMAgentTemplate(
             String jvmOptions,
             RetentionStrategy<?> retentionStrategy,
             boolean executeInitScriptAsRoot,
-            boolean doNotUseMachineIfInitFails
+            boolean doNotUseMachineIfInitFails,
+            String licenseType
```

Comment:
can you add this as a setter annotated with databoundsetter please so its not a breaking change for JCasC users. Then you can revert the changes to the yaml files and this will be good to go.

Reply:
Okay, removed it from the constructor everywhere.